\section{Material and Method}

\subsection{Data Sets}

Cells were taken from tumor of 20 patients diagnosed with glioblastoma, they were  then cultured \textit{in vitro}.
RNA-Seq analysis was then conducted to obtain the raw counts of approximately 20,000 genes for each samples.
Due to licensing limitations, data cannot be shared with the publication and all biological experiments were conducted by the \acrfull{icm} in Paris.
\todo{check the origin of the pdcl dataset}
We will refer to this dataset as \acrfull{pdcl}.
Only one replicate per sample is available.

Because the \acrshort{pdcl} dataset does not includes matching control samples, we compared them to astrocytes RNA-Seq data from a study on the characterization of different astrocytic models by Lundin  al \cite*{Lundin2018}.
Raw counts data are available for download on the \acrfull{geo} platform under the accession number \textbf{GSE109001}.
The dataset contain 4 samples of different astrocyte cell lines with 3 replicates per samples.
The accession number and the name of the samples are listed in table \ref*{table:list-control-samples}.
At the time of the publication, data were last updated on January 29, 2019.

\begin{table}
    \centering
    \begin{tabular}{ |c|c| }
        \hline
        Accession Number & Sample name \\
        \hline
        GSM2927872 & AF22\_NES-Astro\_d29\_Br1 \\
        GSM2927873 & AF22\_NES-Astro\_d29\_Br2 \\
        GSM2927874 & AF22\_NES-Astro\_d29\_Br3 \\
        \hline
        GSM2927881 & iCellAstro\_Br1 \\
        GSM2927882 & iCellAstro\_Br2 \\
        GSM2927883 & iCellAstro\_Br3 \\
        \hline
        GSM2927884 & CCF\_Br1 \\
        GSM2927885 & CCF\_Br2 \\
        GSM2927886 & CCF\_Br3 \\
        \hline
        GSM2927887 & phaAstro\_Br1 \\ 
        GSM2927888 & phaAstro\_Br2 \\ 
        GSM2927889 & phaAstro\_Br3 \\ 
        \hline
    \end{tabular}
    \caption{
        List of samples with their accession number used as the control condition.
        Each samples can be either downloaded separately or as one raw count file in text format using the accession number \textbf{GSE109001} on \acrshort{geo}.
    }
    \label{table:list-control-samples}
\end{table}

We also compare the pathways altered in the \acrshort{pdcl} dataset with the pathways found altered in a dataset accessible on \acrshort{tcga} database. 
We downloaded glioblastoma gene expression data from the TCGA-GBM project for both tumour and control samples.
The lastest \acrshort{tcga} version available \textbf{v33.1} was released on May 31 2022, transcriptomes profilling for all \acrshort{tcga} projects were updated in version \textbf{v32.0} released March 29 2022.
\todo{redo analysis with updated database}
RNA quantification was performed on primary or recurrent tumour for the glioblastoma samples, and on solid tissue for control samples.
Importantly, very few normal transcriptomes profilling are available for control samples in the TCGA-GBM project and none are available for TCGA-LGG (Lower Grade Glioma data).
Thus, this dataset will be over-represented by tumourous samples with 169 tumour samples and 5 control samples.

\subsection{Differential Expression Analysis}

Before we assess the pathways that are altered in case of glioblastoma we need to look for genes whose expression are deregulated.
A \acrfull{de} analysis consist of comparing the expression of a gene, the quantity of RNA produced by a gene, between two conditions (usually disease versus a control).
To quantify the expression of a gene, reads produced during an RNA-Seq experiment are mapped to the genes of a reference genome.
The number of mapped reads is computed for each genes giving a raw value called raw counts.
Factors such as transcript length, total number of reads and sequencing biases can impact the raw counts value. 
Therefore, raw counts need to be normalized to obtain a value suitable for comparison \cite*{Conesa2016}.
Different tools are available to perform \acrshort{de} analysis such as edgeR \cite*{Robinson2010} and Limma \cite*{Ritchie2015}.
In this study, we use DESeq2 \cite*{Love2014} an R package designed for \acrshort{de} analysis.
DESeq2 uses a negative bionomial distribution (also called gamma-Poisson distribution) to model read counts and perform a Wald-Test to test for differential expression.
As DESeq2 includes a built-in normalization method, only raw counts can be used as input to the tool.
It also perform an automated detection of outliers, detection of gene with low counts, estimation of model factors and gene expression dispersion.
The $log_2 fold change$ value is a numeric value used to quantify the strength and direction of deregulation (up or down) between a condition and the control.
We selected DESeq2 for the features cited above, its ease of use and its overall performance \cite*{Love2014}.

\acrshort{de} analysis was performed on the two datasets \acrshort{pdcl} and \acrshort{tcga} where we compared all the tumourous samples against all the normal samples (glioblastoma versus normal).
This will find the mean genes deregulation in the case of glioblastoma or a common glioblastoma profile.
However, 

\subsection{Pathway Enrichment Analysis}

\subsection{Implementation of the Workflow and Interpretation of the Results}

